airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ConnectionPageTitle.tsx

```diff
@@ -1,35 +1,30 @@
 import { faTrash } from "@fortawesome/free-solid-svg-icons";
 import React, { useCallback, useMemo } from "react";
 import { FormattedMessage } from "react-intl";
-import { useNavigate } from "react-router-dom";
+import { useNavigate, useParams } from "react-router-dom";
 
 import { Text } from "components/base/Text";
 import { InfoBox } from "components/InfoBox";
 import StepsMenu from "components/StepsMenu";
 
-import { ConnectionStatus, DestinationRead, SourceRead, WebBackendConnectionRead } from "core/request/AirbyteClient";
+import { ConnectionStatus } from "core/request/AirbyteClient";
+import { useConnectionEditService } from "hooks/services/ConnectionEdit/ConnectionEditService";
 
 import { ConnectionSettingsRoutes } from "../ConnectionSettingsRoutes";
-import ConnectionName from "./ConnectionName";
+import { ConnectionName } from "./ConnectionName";
 import styles from "./ConnectionPageTitle.module.scss";
 import { StatusMainInfo } from "./StatusMainInfo";
 
 interface ConnectionPageTitleProps {
-  source: SourceRead;
-  destination: DestinationRead;
-  connection: WebBackendConnectionRead;
-  currentStep: ConnectionSettingsRoutes;
   onStatusUpdating?: (updating: boolean) => void;
 }
 
-const ConnectionPageTitle: React.FC<ConnectionPageTitleProps> = ({
-  source,
-  destination,
-  connection,
-  currentStep,
-  onStatusUpdating,
-}) => {
+export const ConnectionPageTitle: React.FC<ConnectionPageTitleProps> = ({ onStatusUpdating }) => {
+  const params = useParams<{ id: string; "*": ConnectionSettingsRoutes }>();
```

Comment:
Cut down props to only what's needed.
Previously params was being prop-drilled.